.github/workflows/docker.yml

```diff
@@ -5,6 +5,7 @@ on:
   push:
     branches:
       - master
+      - 'support/*'
```

Comment:
Do we want this to run on support/2.11? But probably no problem as this works AFAIK even though we officially don't support 2.11 in Docker.

Reply:
2.11 doesn’t even have GHA.